catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/searchui/standard/endpoints/WorkspaceBodyReader.java

```diff
@@ -40,7 +43,7 @@ public Map readFrom(
       MediaType mediaType,
       MultivaluedMap<String, String> httpHeaders,
       InputStream entityStream)
-      throws IOException, WebApplicationException {
-    return JsonFactory.create().readValue(entityStream, Map.class);
+      throws IOException {
+    return GSON.fromJson(new InputStreamReader(entityStream), Map.class);
```

Comment:
❓ Who closes the entity stream?

Reply:
This method _appears_ only to be invoked by test code. I will search in more earnest later and confirm that the stream is managed in all cases.